.env.dev.example

```diff
@@ -42,6 +42,19 @@ EMAIL_HOST_PASSWORD=emailpassword123
 EMAIL_USE_TLS=true
 EMAIL_USE_SSL=false
 
+# Set this to true when initializing certbot for domain, false when not
+CERTBOT_INIT=false
```

Comment:
this looks like a merge regression, I think lines 45-46 should be deleted relative to `main`